### PR TITLE
DOC-2069 AA preview content fixes

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: https://redislabs.atlassian.net/browse/DOC-2069
Staged preview: 

- https://docs.redis.com/staging/DOC-2069/kubernetes/active-active/preview/
- https://docs.redis.com/staging/DOC-2069/kubernetes/active-active/preview/prepare-clusters/
- https://docs.redis.com/staging/DOC-2069/kubernetes/active-active/preview/create-reaadb/

### CODE REVIEW

File: content/kubernetes/active-active/preview/create-reaadb.md [hunk 1]
```diff
@@ -103,8 +103,8 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
       name: example-aadb-1
     spec:
```
Comment (alon-zada) [inline content/kubernetes/active-active/preview/create-reaadb.md:104]: imo we should also include the '.spec.databaseSecretName' that sets the AA database password, we are thinking that this should be recommendation (currently this example will create AA DB without password and it is not a good practice), maybe adding a note that this is example w/o the database secret and directing to "how to set database secret" instructions - wdyt?.

**Review by alon-zada — CHANGES_REQUESTED** [review]
Great!
please view one comment.

**Review by nermiller — APPROVED** [review]

File: content/kubernetes/active-active/preview/create-reaadb.md [hunk 2]
```diff
@@ -100,13 +100,15 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
     apiVersion: app.redislabs.com/v1alpha1
     kind: RedisEnterpriseActiveActiveDatabase
     metadata:
-      name: example-aadb-1
```
Comment (alon-zada) [inline content/kubernetes/active-active/preview/create-reaadb.md:103]: let's stay consistent with the same name as in other examples in the doc.
and keep "example-aadb-1"

**Review by alon-zada — APPROVED** [review]
Great!
please view one small comment.

**Review by alon-zada — CHANGES_REQUESTED** [review]
please view one small comment.

File: content/kubernetes/active-active/preview/create-reaadb.md [hunk 3]
```diff
@@ -100,13 +100,15 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
...
+      name: example-reaadb-1
```
Comment (kaitlynmichael) [inline content/kubernetes/active-active/preview/create-reaadb.md:103]:
```suggestion
      name: example-aadb-1
```

**Review by alon-zada — APPROVED** [review]

**Pull request merged**